### PR TITLE
Added Tracking Properties New Connector for failover detector

### PR DESCRIPTION
**Pull request by balavor**
## Changes
- Added `TrackingPixels.Properties` that for now describe all props that are necessary to fire ad engine flow tracker.
- Added `NewConnector` that will work along with the old one until we move everything on a new one. 
- Moved failover detector to the new connector.

@VerizonAdPlatforms/video-partner-sdk-developers: Please review.

### CODE REVIEW

File: sources/VVPSDK.swift [hunk 1]
```diff
@@ -301,6 +301,9 @@ public struct VVPSDK {
             _ = player.addObserver(connector.process)
             _ = player.store.addObserver(with: playerModel, mode: .everyUpdate, connector.process)
             
+            let newConnector = TrackingPixels.NewConnector(reporter: reporter)
```
Comment (VladyslavAnokhin) [inline sources/VVPSDK.swift]: NewConnector - bad name. `PropertiesConnector`, TrackingPropertiesConnector` ..

Comment (AndriiMoskvin) [reply sources/VVPSDK.swift]: Agree!

File: sources/metrics/tracking pixels/TrackingPixelsNewConnector.swift [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+                                      stage: stage,
+                                      width: props.session.playerDimensions?.width,
+                                      height: props.session.playerDimensions?.height,
+                                      autoplay: props.session.isAutoplayEnabled,
+                                      transactionId: props.metaInfo.transactionId,
+                                      adId: props.metaInfo.adVASTId,
+                                      videoViewUID: props.session.playbackSessionId)
+            }
+            
+            /*Ad Failover Detector*/ do {
```
Comment (AndriiMoskvin) [inline sources/metrics/tracking pixels/TrackingPixelsNewConnector.swift]: `guard` and drop comment?

**Comment by balavor** [comment]
@AndriiMoskvin @VladyslavAnokhin RFR

**Review by AndriiMoskvin — APPROVED** [review]

**Pull request merged**